websites/models.py

```diff
@@ -157,7 +166,9 @@ def calculate_checksum(self) -> str:
                 [
                     json.dumps(self.metadata, sort_keys=True),
                     str(self.markdown),
-                    str(self.content_filepath),  # TO DO: Replace with dynamic filepath
+                    self.type,
+                    str(self.dirpath),
+                    str(self.filename),
```

Comment:
Also, it might be good to add `self.title` as well.  This is an edge case, but if I make a really long title - I used `page 1 how long can I make this before bad things start to happen? Let us find out shall we.  Will be interesting.  What could go wrong?` and save, the filename gets truncated to 125 characters (ending with `go-`).  If I then change the title ending from `wrong` to `right`, the checksum doesn't change and github won't be updated with the new title.

Also, not related to this PR, I should have caught it in the previous one, but when I added another page with the same exact long title, I got the error `django.db.utils.DataError: value too long for type character varying(125)` because I think it tried to add a `-1` suffix which pushed it past the character limit.

Reply:
Good catch w/ the checksums. I'll add that

Re: `title`, we only generate the `filename` when the object is first created, and I think we should leave it that way. Eventually we will support updates to the `filename` via the UI, so I think we should avoid auto-updating that value after the `WebsiteContent` object is first created.

Re: character limit, also great catch. I'll look into it

Reply:
Actually, re: title again, we use the title value in the front matter for markdown pages, so we should have been including that value in the checksum to begin with. I'll add it

Reply:
Turns out the issue with `title`, isn't really an edge case after all.  It is included in the file content([example here](https://raw.githubusercontent.com/mattbertorg1/brand-new-website_b7e8517f8e024c3ab556942cb9d36f6a/main/content/page-1-how-long-can-i-make-this-before-bad-things-start-to-happen-let-us-find-out-shall-we-will-be-interesting-what-could-go-.md)), and so if it changes then the checksum should also change to trigger an update to github.